EDDI/MainWindow.xaml.cs

```diff
@@ -988,7 +989,7 @@ private void companionAppClicked(object sender, RoutedEventArgs e)
             }
             else
             {
-                // Logout from the companion app and start again
+                // Logout from the companion EddiApplication and start again
```

Comment:
Aha, "the companion app" refers to the FDevs's defunct iOS app. Hence "the companion app API" or "cAPI" is how we often refer to the API in question.

I would say, "logout from the cAPI and start again"